utils.js

```diff
@@ -0,0 +1,35 @@
+class RecentlyUsedList {
+
+    constructor() {
+        this.items = [];
+    }
+
+    add(newItem) {
+        if (this.items.indexOf(newItem)) {
+            var position = this.items.indexOf(newItem);
```

Comment:
const instead of var.

Reply:
Better to use `const` instead of `var`. ES6. especially since `position` remains unchanged within this scope.